qiskit_ibm_runtime/session.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         service: Optional[QiskitRuntimeService] = None,
         backend: Optional[Union[str, IBMBackend]] = None,
         max_time: Optional[Union[int, str]] = None,
+        session_id: Optional[str] = None,
```

Comment:
This should not be a parameter to the Session constructor, because it implies you can also change other parameters (e.g. `backend`) of an existing session. Instead it should be a class method like `Session.from_id()`

Reply:
My misunderstanding of the intent of this issue. I thought we want the user to be able to supply the `session_id`. But from what you wrote, the intent is to get a particular session using the `session_id`. I will change it.